services/thanos/15.7.23/defaults/cm.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
 data:
   values.yaml: |
     ---
+    image:
+      registry: docker.io
+      repository: bitnami/thanos
+      tag: 0.37.1-debian-12-r0
```

Comment:
Would it be possible to use the `quay.io/thanos/thanos:v0.37.1` image here?

Reply:
As discussed we will use  this image docker.io/bitnami/thanos:0.37.1-debian-12-r0 from bitnami chart.